debian/libopx-system-utils1.install

```diff
@@ -1,2 +1 @@
-usr/lib/*/*.so*
-usr/lib/*/*.so
+usr/lib/*/*.so.*
```

Comment:
This is not going to include the so files with no version suffix.


```
rdatta@xenlogin-eqx-01:/neteng/rdatta/system-utils/pool/stretch-amd64/opx-system-utils$ ls -l libopx-system-utils1_0.1.0_amd64.deb /usr/lib/*.so
-rw-r--r-- 1 rdatta ncore  3262 Nov  8 12:06 libopx-system-utils1_0.1.0_amd64.deb
-rw-r--r-- 1 root   root  39720 Apr 14  2016 /usr/lib/attica_kde.so
lrwxrwxrwx 1 root   root     10 Jan 22  2016 /usr/lib/libau.so -> libau.so.2
lrwxrwxrwx 1 root   root     25 Feb 10  2016 /usr/lib/libganglia.so -> libganglia-3.6.0.so.0.0.0
-rw-r--r-- 1 root   root  27080 Apr 14  2016 /usr/lib/libknotifyplugin.so
lrwxrwxrwx 1 root   root     27 Feb 17  2016 /usr/lib/libxmlsec1-gcrypt.so -> libxmlsec1-gcrypt.so.1.2.20
lrwxrwxrwx 1 root   root     27 Feb 17  2016 /usr/lib/libxmlsec1-gnutls.so -> libxmlsec1-gnutls.so.1.2.20
lrwxrwxrwx 1 root   root     24 Feb 17  2016 /usr/lib/libxmlsec1-nss.so -> libxmlsec1-nss.so.1.2.20
lrwxrwxrwx 1 root   root     28 Feb 17  2016 /usr/lib/libxmlsec1-openssl.so -> libxmlsec1-openssl.so.1.2.20
lrwxrwxrwx 1 root   root     20 Feb 17  2016 /usr/lib/libxmlsec1.so -> libxmlsec1.so.1.2.20

```

Reply:
Unversioned libraries come from the dev package.

https://github.com/open-switch/opx-system-utils/pull/2/files#diff-0ea4544c9cea3b292b01410766187affL1

See: https://lintian.debian.org/tags/non-dev-pkg-with-shlib-symlink.html

Reply:
Also see this: https://lists.debian.org/debian-mentors/2006/03/msg00356.html